medicines/doc-index-updater/src/storage_client/azure_blob_client.rs

```diff
@@ -0,0 +1,129 @@
+use super::{
+    models::{StorageClientError, StorageFile},
+    GetBlob, StorageClient,
+};
+use crate::create_manager::hash;
+use async_trait::async_trait;
+use azure_sdk_core::{
+    BlobNameSupport, BodySupport, ContainerNameSupport, ContentMD5Support, ContentTypeSupport,
+    MetadataSupport,
+};
+use azure_sdk_storage_blob::Blob;
+use azure_sdk_storage_core::client::Client;
+use std::collections::HashMap;
+
+pub struct AzureBlobStorage {
+    pub container_name: String,
+    prefix: String,
+    storage_account: String,
+    master_key: String,
+}
+
+impl Default for AzureBlobStorage {
```

Comment:
I think this is never used, and was originally implemented because clippy said it should be when we had a `new()` implementation. Try removing it unless we have a use case for it. If clippy complains we can turn that warning off.